apps/gameserver/src/Loot.cpp

```diff
@@ -0,0 +1,116 @@
+#include "Loot.h"
+
+using namespace bango::processor;
+
+void LootGroup::set(lisp::var param)
+{
+    switch (FindAttribute(param.pop()))
+    {
+        case A_INDEX: m_group.m_index = param.pop(); break;
+        case IC_MONEY:
+        {
+            while (!param.null()) 
+            {
+                auto values_from_bracket = GetValuesFromBrackets(param);
+                AssignGroup(values_from_bracket);
+            }
+            break;
+        }
+        case A_ITEM:
+        {
+            while (!param.null()) 
+            {
+                auto values_from_bracket = GetValuesFromBrackets(param);
+                AssignGroup(values_from_bracket);
+            }
+            break;
+        }
```

Comment:
```suggestion
        case IC_MONEY:
        case A_ITEM:
        {
            while (!param.null()) 
            {
                auto values_from_bracket = GetValuesFromBrackets(param);
                AssignGroup(values_from_bracket);
            }
            break;
        }
```

Both cases are indentical, merge to one?

Reply:
true, done.